### ISSUE TITLE
`The requested module ... does not provide an export named ...`

### ISSUE THREAD

**Issue by KyGost** [notiflix/Notiflix#52]
The library works fine (really well, thanks!) as a global src but I'm trying to change to using it as a module import but getting errors no matter what I try.

Using 3.2.5, trying to import individual modules or the whole thing, from the full file, or individuals, all throws for some reason.

**Comment by KyGost** [comment]
3.2.0 doesn't work for me either.... not sure what I'm doing wrong:
```js
import {Block} from '../notiflix/notiflix-block-aio-3.2.0.min.js'
```

**Comment by furcan** [comment]
https://github.com/notiflix/Notiflix#a-install-and-import

**Comment by KyGost** [comment]
> https://github.com/notiflix/Notiflix#a-install-and-import

Is that to say that vanilla modules aren't supported...?

**Comment by KyGost** [comment]
I'll take lack of reply as a yes.

I'll make a new issue relating to that.

**Comment by KyGost** [comment]
Resolved via #54
